front-end-web/src/Components/Pages/MainPage.tsx

```diff
@@ -555,6 +567,11 @@ const MainPage: React.FunctionComponent = () => {
         open={notification.open}
         update={handleChangeNotification}
       />
+      <ProfilePage
+        openPage={profilePage}
+        handleToggleProfile={handleToggleProfile}
+        handleNewNotification={handleChangeNotification}
```

Comment:
I would add `currentUser={currentUser}` here. The current user is already stored in the React state, so it is faster to get from the state than it is from Firebase each time.